include/directory.php

```diff
@@ -24,3 +25,17 @@ function directory_run(&$argv, &$argc){
 	}
 	return;
 }
+
+function directory_update_all() {
+	$r = q("SELECT `url` FROM `contact`
+		INNER JOIN `profile` ON `profile`.`uid` = `contact`.`uid`
+		INNER JOIN `user` ON `user`.`uid` = `contact`.`uid`
+			WHERE `contact`.`self` AND `profile`.`net-publish` AND `profile`.`is-default` AND
+				NOT `user`.`account_expired` AND `user`.`verified`");
+
+	if (dbm::is_result($r)) {
+		foreach ($r AS $user) {
+			proc_run(PRIORITY_LOW, 'include/directory.php', $user['url']);
```

Comment:
This is likely to start hundreds of processes, how is the capacity overflow handled now that we removed the worker queue?